pkg/components/istio-operator/component.go

```diff
@@ -88,6 +88,10 @@ func (c *component) Metadata() components.Metadata {
 		Name: name,
 		Namespace: k8sutil.Namespace{
 			Name: namespace,
+			Labels: map[string]string{
+				"istio-operator-managed": "Reconcile",
+				"istio-injection":        "disabled",
+			},
```

Comment:
Istio added istio operator namespace config. And it was added with above labels.